ste/mgr-JobPartTransferMgr.go

```diff
@@ -327,7 +328,11 @@ func (jptm *jobPartTransferMgr) Info() TransferInfo {
 				 * we can have 4 blocks in core, waiting for a disk or n/w operation. Any higher block size would *sort of*
 				 * serialize n/w and disk operations, and is better avoided.
 				 */
-				blockSize = sourceSize / common.MaxNumberOfBlocksPerBlob
+				if (sourceSize % common.MaxNumberOfBlocksPerBlob == 0) {
```

Comment:
Nit: Using `ceil` function.

Reply:
For ceil to work it should be float. I think you mean this way blockSize = math.Ceil(float64(sourceSize)/common.MaxNumberOfBlocksPerBlob)